### PR TITLE
Add support for spaces query status parameter  

### PR DESCRIPTION
**Pull request by gogo-jscott**
**Before submitting a pull request,** please make sure the following is done:

1. Fork [the repository](https://github.com/availity/availity-react) and create your branch from `master`.
2. Run `yarn` in the repository root.
3. If you've fixed a bug or added code that should be tested, add tests!
4. Ensure the test suite passes (`yarn test`). Tip: `yarn test --watch TestName` is helpful in development.
5. Make sure your code passed the conventional commits check. Read more about [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)

### CODE REVIEW

File: packages/page-header/src/PageHeader.js [hunk 1]
```diff
@@ -165,6 +167,7 @@ PageHeader.propTypes = {
   clientId: PropTypes.string,
   iconSrc: PropTypes.string,
   iconAlt: PropTypes.string,
+  variables: PropTypes.object,
```
Comment (jordan-a-young) [inline packages/page-header/src/PageHeader.js]: Can you add this to the PageHeaderProps type as well please? The type should be `Record<string, unknown>`

**Review by jordan-a-young — CHANGES_REQUESTED** [review]
I think it looks good other than the missing type def.

@chrishavekost @owolfhu1 @gregmartDOTin Will updating the default query have any negative impact on the spaces api?

File: packages/page-header/src/PageHeader.js (hunk 1 above)
Comment (jordan-a-young) [reply packages/page-header/src/PageHeader.js]: Also, what do you think about naming the prop `spacesVariables`? That way it will be less generic

**Comment by chrishavekost** [comment]
I think @owolfhu1 is looking into how this will affect the API but I'm wondering if @gogo-jscott can explain the reasoning behind these changes and if they're needed for everyone. If it's a one-off change I think each component will let you override the default query

**Comment by dnoler** [comment]
@chrishavekost I can provide context for this change. We source payer spaces for payer logos, but we have an issue on Claims where a particular payer has an inactive payer space, so their logo isn't found. Granted, we probably should have a better solution for how we source payer logos, but this gets us to where we can at least define inactive spaces for payers that don't have a proper payer space section on the portal.

**Comment by owolfhu1** [comment]
Can confirm updated query is still working

**Comment by gogo-jscott** [comment]
Closing since feature goes outside intended purpose of package.

**Pull request closed without merging**